### ISSUE TITLE
Help with recording transfer

### ISSUE THREAD

**Issue by Furiouss38** [blindsidenetworks/scalelite#229]
Hi,
I actually have in production a standalone BBB server.
Because of upcoming exams, some perf issues and more and more people wanting to use it, we decided to install a scalelite cluster.
So i've installed my scalelite, nfs and 3 bbb servers.
Problem is :
my actual bbb is in 2.0, my cluster is in 2.2.
I want to transfer the actual published recordings without adding the old bbb server in the cluster is it possible ?
I thought about transfering the files in the nfs folder and reuse the same bbb-secret on one of the members, would it be enought to user the batch import script ?

**Comment by Furiouss38** [comment]
Anyone ? :(

**Comment by avkovalevs** [comment]
You can upgrade old BBB to 2.2 and add it to the cluster. After that move published records to NFS.

**Comment by farhatahmad** [comment]
I don't believe that you actually need to add the server to the cluster for you to move the recordings. If you set up the shared volume and run the batch importer script, it should move all your recordings from that server to scalelite

**Comment by Furiouss38** [comment]
Ok, i'll try that next day of work, keep you posted

**Comment by Furiouss38** [comment]
Ok, I succesfuly ran the bash script on my old bbb where i mounted the shared nfs volume.
/data/scalelite-recordings/var/bigbluebutton/spool/
Some records where imported and disappeared from /data/scalelite-recordings/var/bigbluebutton/spool/  and I can access them with  https://<my scalelite front>/playback/presentation/2.0/playback.html?meetingId=<meeting id> 
But since 1h the list of file in that folder is not moving anymore and I don't know where to find logs or anything to help me find out why ?

**Comment by Furiouss38** [comment]
I found the logs with "docker logs scalelite-recording-importer"
For the few recordings that where not imported there are errors : 

```
E, [2020-06-02T08:39:45.466209 #6] ERROR -- : Failed to import recording: bad URI(is not URI?): "\n      https://bbb/playback/presentation/2.0/playback.html?meetingId=1231231231321\n    "
W, [2020-06-02T08:39:45.466626 #6]  WARN -- : /usr/lib/ruby/2.6.0/uri/rfc3986_parser.rb:67:in `split': bad URI(is not URI?): "\n      https://bbb/playback/presentation/2.0/playback.html?meetingId=1231321321321321\n    " (URI::InvalidURIError)
```
(the URL here are fake ids and hostname but they're good in the real file")

I don't understand why since other meetings who worked have the same URL type in metadata
And even some of them older from an older older install have URL that are not working anymore but the import is doing well.

**Comment by farhatahmad** [comment]
Are you running the latest version? I believe that was a bug fixed in v1.0.2
